### PR TITLE
Update sonar-dependency-check-plugin to 2.0.4

### PR DESCRIPTION
**Pull request by Reamer**
We've released sonar-dependency-check-plugin version 2.0.4, please add it to the marketplace.
Thanks in advance!

### CODE REVIEW

**Comment by Reamer** [comment]
https://community.sonarsource.com/t/new-release-sonar-dependency-check-plugin-2-0-4/22140

File: dependencycheck.properties [hunk 1]
```diff
@@ -1,13 +1,19 @@
...
 description=Integrates Dependency-Check reports into SonarQube
 homepageUrl=https://github.com/dependency-check/dependency-check-sonar-plugin
-archivedVersions=2.0.2
-publicVersions=2.0.3
+archivedVersions=2.0.2,2.0.3
+publicVersions=2.0.4
 defaults.mavenGroupId=org.sonarsource.owasp
 defaults.mavenArtifactId=sonar-dependency-check-plugin
 
+2.0.4.description=Integrates Dependency-Check reports into SonarQube
```
Comment (ganncamp) [inline dependencycheck.properties]: You can't have the same description for every release. This value tells users what changed in _this version_ so they can judge whether they want the upgrade.

**Comment by Reamer** [comment]
@ganncamp Can you retrigger travis-ci. I think [GitHub <-> travis issue](https://www.traviscistatus.com/incidents/rx6fhs3wqcln) isn't solved.

**Comment by ganncamp** [comment]
@Reamer I approved. I didn't merge. :roll_eyes:

**Pull request merged**